spesifikasjoner/afpant/afpant-kjoepekontrakt/kjoepekontrakt-teknisk-beskrivelse.md

```diff
@@ -0,0 +1,152 @@
+
+### Negativt resultat (NACK)
+- Tom payload returneres (ZIP arkiv med dummy innhold). Manifest key "status" og "statusDescription" må avleses for årsak.
+
+# Meldingstype: KjoepekontraktFraMegler
+Megler sender uoppfordret til bank eller ved ved signifikante endringer.
+
+* **Sendes uoppfordret** (bank har ikke forespurt kjøpekontrakt)  
+  Denne meldingen kan kun sendes uoppfordret fra megler til bank dersom kjøpekontrakten er signert. 
+Den signert kjøpekontrakten skal da finnes i ZIP-arkivet som _signert_kjoepekontrakt*.(pdf|sdo)_ 
+OG metdata skal inneholde en ressurs/vedlegg med korrekt navn.
+* **Ved endring** (bank har tidligere forespurt kjøpekontrakt eller mottatt signert kjøpekontrakt uoppfordret)  
+  Sendes ved endring, dvs. at megler har tidligere sendt uoppfordret eller har mottatt en forespørsel.
+Felter som krever at det sendes en `KjoepekontraktFraMegler` melding til bank er marker med _hake_ i tabell i _hake_ i tabellen i [løsningsbeskrivelsen](kjoepekontrakt-loesningsbeskrivelse.md#svar-fra-megler)
+
+Dersom kjøpekontrakten er signert skal vedlegget med korrekt filnavn være definert som en ressurs/vedlegg i meldingens metadata OG eksistere i ZIP-arkivet.
+
+_Denne meldingen skal ikke besvares._ 
+
+## Validering og ruting(banksystem)
+Mottakende systemleverandør søker blant lånesaker hvor saksnummer matcher `kjoepekontrakt.bank.referanse` dersom den er angitt.
+
+Dersom dette er en uoppfordret melding med signert kjøpekontrakt, trenger ikke `kjoepekontrakt.bank.referanse` være utfylt.
+I disse tilfellene må bank:
+* søke blant lånsesaker avgrenset til:
+  * eiendommer/borettsandeler angitt i kjøpekontrakten
+  * OG minst 1 kjøper angitt i kjøpekontrakten
+  
+## Validering og ruting(meglersystem)
+Megler må sette `kjoepekontrakt.bank.referanse` dersom megler tidligere har mottatt [KjoepekontraktforespoerselFraBank](#meldingstype-kjoepekontraktforespoerselfrabank) 
+
+## Payload
+At ZIP-arkiv som inneholder en xml-fil med **KjoepekontraktFraMegler** som root element og som er i henhold til [definert skjema](../afpant-model/xsd/dsve-1.0.0.xsd).
+Dersom kjøpekontrakten er signert skal ZIP-arkiv også innholde den signert kjøpekontrakten være med som en egen fil.
+Den signert kjøpekontrakten er enten en PDF eller en SDO.
+
+**Krav til filnavn i ZIP-arkiv:**
+* Filnavnet til meldingen kjoepekontraktFraMegler må følge konvensjonen: _kjoepekontraktframegler*.xml_ . (navnet til meldingen i lowercase)
+* Filnavnet til en evt. signert kjøpekontrakt skal følge konvensjonen: _signert_kjoepekontrakt*.(pdf|sdo)_.
+
+## Manifest
+(BrokerServiceInitiation.Manifest.PropertyList)
+
+|Manifest key|Type|Required|Beskrivelse|
+|--- |--- |--- |--- |
+|messageType|String|Yes|KjoepekontraktFraMegler|
+
+# Fotnoter
+<b id="f1">1</b> Aktive oppdrag [↩](#ref_f1)
+* Når "arkiveres" eller ferdigstilles et oppdrag? Dette er opp til megler/meglersystem.
```

Comment:
er dette noe som kan avklares etter merge av PR eller bør vi ta det i forkant? er det her snakk om kritierier for å filtrere ut gjeldende kjøpekontrakter som skal returneres til bank for et bestemt kjøper fødselsnr eller orgnr? (dvs arkiverte/ferdigstilte oppdrag utelates fra returdata)